.gitmodules

```diff
@@ -1,3 +1,6 @@
 [submodule "firefox-ui-tests"]
 	path = firefox-ui-tests
 	url = https://github.com/mozilla/firefox-ui-tests.git
+[submodule "browsermob-proxy"]
```

Comment:
Could you verify that this submodule is actually necessarily? I can't verify locally because extra environment setup is needed.